### PR TITLE
Improving EN translation

### PR DESCRIPTION
**Pull request by lze3**
Have better formality and legibility. Also be more specific and improve grammar in some of the options.

### CODE REVIEW

**Comment by lze3** [comment]
Fixed conflicts.

File: config.lua [hunk 1]
```diff
@@ -52,38 +52,38 @@ Config.TextFr = {
...
-	ban           = "Ban one online player",
-	banoff        = "Ban one offline player",
-	dayhelp       = "Number of day",
-	reason        = "Reason of ban",
-	history       = "Show all ban of a player",
-	reload        = "Reload BanList and BanListHistory",
-	unban         = "Remove one ban from the list",
-	steamname     = "(Steam Name)",
+	ban           = "Ban a player",
+	banoff        = "Manually ban a (offline) player",
```
Comment (RedAlex) [inline config.lua]: Manually are not need to be write.
It's just an in game help to say what the command do.
"Ban an offline player" Will be better no?

**Review by RedAlex — COMMENTED** [review]
Line 82 to be more precise

File: config.lua (hunk 1 above)
Comment (lucmen) [inline config.lua]: banoff     = "ban an offline player",

**Review by lucmen — CHANGES_REQUESTED** [review]

**Review by RedAlex — APPROVED** [review]

**Pull request merged**